index.html

```diff
@@ -416,65 +416,172 @@ <h2 id="liaisons">
+      since all contributions will be tracked via Github mechanisms 
+      (e.g. pull requests). For the same reason, the Community Group 
+      mailing list must not be used for making or discussing substantive contributions
+      to Specifications.
     </p>
+    
+    <h2 id="transparency">
+      Transparency
+    </h2>
+    
     <p>
-      Note: this Community Group will not use a <em>contrib</em> mailing list
-      for contributions since all contributions will be tracked via Github
-      mechanisms (e.g. pull requests).
+      The group will conduct all of its technical work in public. If the group 
+      uses GitHub, all technical work will occur in its GitHub repositories 
+      (and not in mailing list discussions). This is to ensure contributions 
+      can be tracked through a software tool.
     </p>
+  
+   <p class="note">Proposed charter deleted this paragraph</p>  
+    <p>
+      Meetings may be restricted to Community Group participants, but a 
+      public summary or minutes must be posted to the group's public mailing 
+      list, or to a GitHub issue if the group uses GitHub.
+    </p>
+
+    <h2 id="decision">
+      Decision Process
+    </h2>
+    
+    <p>This group will seek to make decisions where there is consensus. 
+    Groups are free to decide how to make decisions (e.g. Participants 
+    who have earned Committer status for a history of useful contributions 
+    assess consensus, or the Chair assesses consensus, or where consensus 
+    isn't clear there is a Call for Consensus [CfC] to allow multi-day online 
+    feedback for a proposed course of action). It is expected that participants 
+    can earn Committer status through a history of valuable contributions as is 
+    common in open source projects. After discussion and due consideration of 
+    different opinions, a decision should be publicly recorded (where GitHub is 
+    used as the resolution of an Issue).
+    </p>
+    
+    <p>
+      If substantial disagreement remains (e.g. the group is divided) 
+      and the group needs to decide an Issue in order to continue to make progress, 
+      the Committers will choose an alternative that had substantial support 
+      (with a vote of Committers if necessary). Individuals who disagree 
+      with the choice are strongly encouraged to take ownership 
+      of their objection by taking ownership of an alternative fork. 
+      This is explicitly allowed (and preferred to blocking progress) 
+      with a goal of letting implementation experience inform which 
+      spec is ultimately chosen by the group to move ahead with.
+    </p>
+    
+    <p>
+      Any decisions reached at any meeting are tentative and should be 
+      recorded in a GitHub Issue for groups that use GitHub and otherwise 
+      on the group's public mail list. Any group participant may object to 
+      a decision reached at an online or in-person meeting within 7 days of 
```

Comment:
@anssiko The Second Screen Working Group uses a 10 day window for decisions made in face to face meetings, and I believe there was a feeling in the room to use the same process for the CG.  Anssi do you have thoughts?